web/src/components/storage/ProposalSettingsSection.jsx

```diff
@@ -40,6 +41,18 @@ import SpacePolicyField from "~/components/storage/SpacePolicyField";
  * @typedef {import ("~/client/storage").Volume} Volume
  */
 
+/**
+ * A helper function to decide whether to show the progress skeletons or not
+ * for the specified component
+ * @param {boolean} loading loading status
+ * @param {string} component name of the component
+ * @param {symbol} changing the item which is being changed
+ * @returns {boolean} true if the skeleton should be displayed, false otherwise
+ */
+const ShowSkeleton = (loading, component, changing) => {
```

Comment:
I would not use PascalCase here. It looks like a component but actually is not a component.